manifests/director.pp

```diff
@@ -6,9 +6,22 @@
 
   validate_re($title,'^[A-Za-z0-9_]*$', "Invalid characters in director name ${title}. Only letters, numbers and underscore are allowed.")
 
+  $template_director = $::varnish::params::version ? {
+    4       => 'varnish/includes/directors4.vcl.erb',
+    default => 'varnish/includes/directors.vcl.erb',
+  }
+
+  if $template_director == 'varnish/includes/directors4.vcl.erb' {
```

Comment:
It would make more sense to check `$::varnish::params::version` directly here. As the version has just been checked to select the correct template, it might be even clearer do all this in one big `if` or `case` statement.